blivet/formats/swap.py

```diff
@@ -21,8 +21,10 @@
 #
 
 from parted import PARTITION_SWAP, fileSystemType
+from ..errors import FSWriteUUIDError
```

Comment:
Just a note on the commit message -- swap is not a file system. It's just a device format, similarly to LVM PV, LUKS or these things.